bundle.yaml

```diff
@@ -1,19 +1,22 @@
----
+bundle: kubernetes
+name: mlflow
+description: "MLflow"
+
 applications:
   mlflow-server:
-    charm: "cs:mlflow-server"
+    charm: "ch:mlflow-server"
+    channel: "latest/edge"
```

Comment:
Should we add [this](https://github.com/canonical/argo-operators/pull/17/files#diff-73c670418fb192fb77d3a1c95ada8448ee256f1a6afa0e6ede380a7c63221184R15)?